manifests/p/Pallhed/LiveresultatDesktop/1.1.0/Pallhed.LiveresultatDesktop.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: Pallhed.LiveresultatDesktop
+PackageVersion: 1.1.0
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://downloads.pallhed.se/LiveresultatDesktopSetup1.1.0.exe
+  InstallerSha256: B2F1227BFCC6D4B94BB7E16EB05587068FE86BDD91C0B38572D89AE1F5E1DE1A
+  InstallerSwitches:
+    Custom: /norestart /verysilent /allusers

```

Comment:
`/verysilent` should be used for the `Silent:` switch rather than the custom one
`Scope: machine` should be specified when using `/allusers`
An additional installer node for `Scope: user` should be added with the switch `/currentuser` if supported